src/GitHub.Exports.Reactive/Models/IInlineAnnotationModel.cs

```diff
@@ -0,0 +1,23 @@
+﻿namespace GitHub.Models
+{
+    /// <summary>
+    /// Represents an inline annotation on an <see cref="IPullRequestSessionFile"/>.
+    /// </summary>
+    public interface IInlineAnnotationModel
```

Comment:
Is this interface needed? `InlineAnnotationModel` is an immutable class so there's nothing to mock really.

Reply:
Fixed